cabot/cabotapp/models.py

```diff
@@ -334,6 +334,18 @@ class StatusCheck(PolymorphicModel):
         default=True,
         help_text='If not active, check will not be used to calculate service status and will not trigger alerts.',
     )
+    activity_counter = models.PositiveIntegerField(
+        default=0,
+    )
+    use_activity_counter = models.BooleanField(
+        default=False,
+        help_text='When enabled, a check\'s \'activity counter\' is used to tell if '
+                  'the check can run. The activity counter starts at zero and may be '
+                  'incremented or decremented via API call. When incremented above '
+                  'zero, the check may run.  When decremented to zero, the check will '
+                  'not run. This allows external processes to enable or disable a check '
+                  'as needed. (Note: the check must also be marked \'Active\' to run.)',
```

Comment:
Great explanation, was kind of worried about people being confused about this new field but I think this definitely gives them enough information about it. 

As we continue adding new fields to status checks I think we should definitely try to figure out some kind of "advanced options" flow to simplify things. I'll ticket it